src/main/resources/schemas/internal-dar-notify-event.xsd

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
2 very similar schemas in this folder. If both are needed, maybe add a comment to this file to make clear to a future developer why we need both and what the difference is.